app/controllers/root_controller.rb

```diff
@@ -1,4 +1,3 @@
 class RootController < ApplicationController
-  def index
-  end
+  def index; end
```

Comment:
- [ ] Great effort working on your `Hello-rails` using one app. But I notice some of the requirement for the project milestone has not been implemented yet. to be sincere you still have some more to do in order to achieve the purpose of this mile. I would kindly suggest you stick to the project requirement as much as possible. As this will help a long way to assist you in achieving your goal.

Below is the requirement 👇 

<img width="773" alt="1" src="https://github.com/mrkamin/Hello-Rails-React/assets/78634841/3e0eae9c-7c11-4afa-96b2-66a482921508">
<img width="770" alt="2" src="https://github.com/mrkamin/Hello-Rails-React/assets/78634841/5283aa7f-db61-4efe-a703-8b30e0e8a11b">

Reply:
@brainconnect93 
thanks for your good and valuable feedback. your suggestions applied.